src/ImageProcessorCore/Colors/ColorTransforms.cs

```diff
@@ -18,9 +19,60 @@ namespace ImageProcessorCore
     public partial struct Color
     {
         /// <summary>
+        /// Blends two colors by lightening.
+        /// </summary>
+        /// <remarks>
+        /// Selects the lighter of the backdrop and source colors.
+        /// The backdrop is replaced with the source where the backdrop is lighter; otherwise, it is left unchanged
+        /// Note: This behavior is on a channel by channel basis, i.e., this rule is applied to each of the 3 RGB color channels separately.
+        /// </remarks>
+        /// <param name="backdrop"></param>
+        /// <param name="source"></param>
+        /// <returns>
+        /// The <see cref="Color"/>.
+        /// </returns>
+        public static Color Lighten(Color backdrop, Color source)
+        {
+            var newVector = new Vector4
```

Comment:
:8ball: I must be misunderstanding something about pre/post-multiplied alpha. I was under the impression that for all intents and purposes it could be represented as: 

Pre-multiplied: R,G,B,A
Post-multiplied: (R*A),(G*A),(B*A),A
:.
Pre-multiplied red w/ 50% opacity: 255, 0, 0, .5
Post-multiplied red w/ 50% opacity: 127.5, 0, 0, .5

Where pre-mult is the 'true color' and post-mult is the 'color after multiplication by alpha'. Such that we keep the same alpha value in post and pre, but the RGB is different. We can convert from post to pre by correcting for the alpha, but if we don't know whether or not a color X is post or pre then we can't do anything without potentially effing it up.